Clients/src/presentation/pages/EvalsDashboard/ExperimentDetailContent.tsx

```
@@ -1027,83 +1092,166 @@ export default function ExperimentDetailContent({ experimentId, projectId, onBac
+                                    color: isUser ? "#059669" : "#1E40AF",
+                                    display: "block",
+                                    mb: 0.5,
+                                    fontSize: "10px",
+                                    textTransform: "uppercase",
+                                  }}
+                                >
+                                  {isUser ? "User" : "Assistant"}
+                                </Typography>
+                                <Box
+                                  sx={{ fontSize: "12px", color: "#374151" }}
+                                  dangerouslySetInnerHTML={{ __html: markdownToHtml(turn.content || "") }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)